client/gui/CGuiHandler.cpp

```diff
@@ -192,7 +192,7 @@ void CGuiHandler::handleEvents()
 	while(!SDLEventsQueue.empty())
 	{
 		continueEventHandling = true;
-		SDL_Event ev = SDLEventsQueue.front();
+		SDL_Event& ev = SDLEventsQueue.front();

```

Comment:
Change of behavior - after `SDLEventsQueue.pop` variable `ev` will now point to different event since it is now a reference.

Yes, I agree that code is at least weird, if not outright broken, but changed version is very likely to be broken as well and probably even worse than before.

If this is necessary for build - can you at least change it to static? Not sure whether this will work though sice I don't know what initial warning was.
```suggestion
		static SDL_Event ev = SDLEventsQueue.front();

```

Reply:
@IvanSavenko done static